src/Employee.java

```diff
@@ -0,0 +1,47 @@
+public class Employee {
+    private final String fullName;
+    private int department;
+    private float salary;
+    private final int id;
+    static int counter = 0;
+
+    public static int id() {
+        counter++;
+        return counter;
+    }
+    public Employee (String fullName, int department, int salary) {
+        this.id = id();
```

Comment:
```suggestion
        this.id = counter++;
```
и всё, никаких методов ненужных